platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/InputValidation.java

```diff
@@ -97,4 +105,22 @@ public static boolean checkForClientSideVulnerableMimeType(String mimetype) {
         }
         return true;
     }
+
+    /**
+     * Checks for filenames that have been disallowed by the system.
+     *
+     * @param filename
+     * @return true if the filename is acceptable
+     */
+    public static boolean checkForClientSideVulnerableFileName(String filename) {
```

Comment:
✏️  Might this method name benefit from following the naming convention of starting with a form of "to be", as in `isClientSideVulnerableFileName`? Might help with readability by future developers where this is used so they know it is a boolean return.